.eslintrc.json

```
@@ -34,7 +33,7 @@
     "no-undef": "off",
     "@typescript-eslint/explicit-function-return-type": "off",
     "@typescript-eslint/explicit-module-boundary-types": "off",
-    "@typescript-eslint/no-use-before-define": "warn",
+    "@typescript-eslint/no-use-before-define": "off",
```

```suggestion
    "@typescript-eslint/no-use-before-define": ["warn", { functions: false }],
```
https://eslint.org/docs/latest/rules/no-use-before-define#options

---

agree with Dynst here @hippocampus-web3

---

I changed it but applied the same configuration that reacts-scripts was using to avoid new issues and having to combine a code refactor with the migration of the transpilation tool in the same PR. 

https://github.com/facebook/create-react-app/blob/6254386531d263688ccfa542d0e628fbc0de0b28/packages/eslint-config-react-app/index.js#L63